code/game/objects/items/tools/shovel_tools.dm

```diff
@@ -171,7 +177,27 @@
 		force = 30
 	..()
 
+/obj/item/tool/shovel/etool/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/weapon/claymore/mercsword/machete))
+		return ..()
+	if(sharpened)
+		to_chat(user, "<span class='notice'>The entrenching tool is already sharpened.</span>")
+		return
+	if(folded)
+		to_chat(user, "<span class='notice'>You cannot sharpen the entrenching tool while it is folded.</span>")
+		return
+	user.visible_message("<span class='notice'>[user] begins to sharpen the [src].</span>",
+	"<span class='notice'>You begin to sharpen the [src].</span>")
+	if(!do_after(user, 2 SECONDS, TRUE, src, BUSY_ICON_FRIENDLY))
+		return
+	sharpened = TRUE
+	name = "sharpened" + name
```

Comment:
```suggestion
	name = "sharpened " + name
```

Or maybe not, check that it's not needed

Reply:
name doesnt have a space in front of it, so the space means it will  be sharpened etool not sharpenedetool. at least thats how im assuming it works.

Reply:
yeah, and it should be sharpened etool  and not sharpenedetool

Reply:
again im big dumb, i missread git once more